v2/containeranalysis/client.go

```diff
@@ -194,6 +194,9 @@ func NewClient(ctx context.Context, binauthProject string, keyring signer.Attest
 	if err != nil {
 		return nil, err
 	}
+	if err := caClient.Close(); err != nil {
```

Comment:
could we use defer here to make extra sure we only close the connection once we're done using it?
`defer caClient.Close()`

Reply:
We'd lose the ability to react to the returned `err` (aside from logging), which I kind of liked.

I think this is moot: rather than create the `containeranalysis.NewClient` just to crack it open and steal the `grafeas.NewClient` it creates, we should just call `grafeas.NewClient` directly!

I'm going to dismiss review as that's a 180 from what you approved, but I think it addresses this concern by removing the `caClient` entirely.

Reply:
I like this idea! if we're not using the containeranalysis client for anything other than calling functions on the grafeas client, we can just use it directly